horton/scripts/wpart.py

```diff
@@ -119,7 +119,8 @@ def wpart_slow_analysis(wpart, mol):
     operators = []
     for ipure in xrange(npure):
         for iatom in xrange(wpart.natom):
-            operators.append(wpart.cache.load('overlap_operators', iatom)['olp_%05i' % ipure])
+            operators.append(wpart.cache.load('overlap_operators', iatom)[
+                             'olp_%05i' % ipure])
 
```

Comment:
From Travis: `C0330 Wrong hanging indentation (remove 13 spaces).`
I would do this differently that pylint suggests.
1) It is ok best to just put it just back on one line.
2) If you want to wrap, put the square bracket also on the second line, i.e. wrap after `iatom)` not after `iatom)[`.